test/changelog.test.js

```diff
@@ -0,0 +1,24 @@
+const assert = require("assert");
+const {spawn} = require("child_process");
+const fs = require("fs");
+const transform = require("../lib/changelog.js");
```

Comment:
You currently aren't using this - do you want to keep it to check that this file exists?

Reply:
This is called by mocha, so it is used.

Reply:
The tests still run if this line is commented out or removed, and ESLINT complains if it is left in.

Reply:
You meant the `transform`, I deleted that.